src/services/anchor-service.ts

```diff
@@ -123,22 +135,22 @@ export default class AnchorService {
 
     // create anchor records on IPFS
     logger.debug("Creating anchor commit")
-    const anchors = await this._createAnchorCommits(ipfsProofCid, merkleTree, requests);
+    const anchors = await this._createAnchorCommits(ipfsProofCid, merkleTree);
 
     // Update the database to record the successful anchors
     logger.debug("Persisting results to local database")
-    await this._persistAnchorResult(anchors)
+    await this._persistAnchorResult(anchors, acceptedRequests)
 
     logEvent.anchor({
       type: 'anchorRequests',
       requestIds: requests.map(r => r.id),
-      clashingRequestsCount: numInitialRequests - requests.length,
-      validRequestsCount: requests.length,
+      clashingRequestsCount: requests.length - acceptedRequests.length, // This field is poorly named
```

Comment:
Note @v-stickykeys, this pr changes the meaning of this `clashingRequestsCount` field so it's now kind of poorly named.  Previously it only reported requests rejected by conflict resolution, but now it will return a count of requests that failed for *any* reason including conflict resolution, networking errors, timeouts, etc.

I can rename the field but was afraid that would break the metrics dashboards - let me know if you think I should go ahead and rename it to `failedRequestsCount`

Reply:
We aren't tracking this in metrics anyway right now but the best thing to do is separate them and name them accordingly. So that would be 
```
conflictingRequestsCount
totalFailedRequestsCount
acceptedRequestsCount
```

etc..

Reply:
sweet thanks, updated